### DISCUSSION TITLE
AI based physics/enginering prototyping and design company

### DISCUSSION THREAD

**Discussion by RiteshBalayan** [RiteshBalayan/instaorbit#7] [Ideas]
It will be very clean differentiator for us.
Making us unique from other simulation companies and broaden us from Space sector.

**Comment by hrithikthakur** [comment]
so, space mission management is the our starting market, and this is the broader mission of the company. this makes me wonder how it would affect marketing. because if you only market yourslef as a niche company doing a niche thing, you can sell faster, but your market is small.

**Reply by RiteshBalayan** [reply]
Keep broader mission for prime investors like YC and not confuse customers and small investors.

**Reply by hrithikthakur** [reply]
but YC will also look if we've our beachhead market segment. I think this needs to be triangualted with an expert. should add to the meeting agenda with one of the experts frmo eagle labs
